server/index.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import mongoose from "mongoose";
+import cors from "cors";
+import dotenv from "dotenv";
+import bodyParser from "body-parser";
+import loginRoute from "./routes/logIn.js";
+import signUpRoute from "./routes/signUp.js";
+
+const app = express();
+dotenv.config();
+
+// set them to properly send a reqest
+app.use(bodyParser.json({ limits: "30mb", extended: true }));
+app.use(bodyParser.urlencoded({ limits: "30mb", extended: true }));
```

Comment:
don't need `bodyParser` anymore. https://expressjs.com/en/api.html
`express.json()` can perse incoming request as itself in Express v4.16.0 onwards.

@miyabitanimchi did you get an error without it?